implementer-guide/user-management/guest-user.md

```diff
@@ -1,37 +1,41 @@
 ---
 title: Guest user
 page_title: Guest user
-description: Guest user
+description: Explains the Guest user feature and its purpose and limitations
 slug: guest-user
-tags: users,guest
+tags: users,guest,share,embed
 published: True
 position: 400
 ---
 
 # Guest user
+The Guest user is a built-in user account That is created upon the initial setup of the Telerik Report Server. The Guest's purpose is to be used for report document preview and sharing, either using a sharing link to a report, a report viewer control embedded in a client application, or the reports API.
```

Comment:
"... user account That is created..." => lowercase on "That"?

Reply:
Shame. Fixed